app/src/main/java/com/example/bigowlapp/activity/SupervisedGroupListActivity.java

```diff
@@ -1,59 +1,103 @@
 package com.example.bigowlapp.activity;
 
+import android.content.Context;
 import android.content.Intent;
 import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
 import android.widget.ArrayAdapter;
 import android.widget.ListView;
-
-import androidx.appcompat.app.AppCompatActivity;
-import androidx.lifecycle.LiveData;
-import androidx.lifecycle.ViewModelProvider;
+import android.widget.TextView;
 
 import com.example.bigowlapp.R;
 import com.example.bigowlapp.model.Group;
 import com.example.bigowlapp.viewModel.SupervisedGroupListViewModel;
-import com.google.firebase.auth.FirebaseUser;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.lifecycle.LifecycleOwner;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.ViewModelProvider;
+
 public class SupervisedGroupListActivity extends AppCompatActivity {
-    private ListView lv;
-    private SupervisedGroupListViewModel supervisedGroupListViewModel;
-    private LiveData<List<Group>> listOfGroupsLiveData;
+    private ListView supervisedGroups;
+    private static SupervisedGroupListViewModel supervisedGroupListViewModel;
```

Comment:
You don't need static for this object.

Note: When you remove static here, remove static for the SupervisedGroupAdaptor class that you created below. The compiler will probably complain if you don't, since non-static field can't be in static classes.

Reply:
Got it! Thanks for pointing it out!